### PR TITLE
Update readme about who does site changes

### PR DESCRIPTION
**Pull request by benjaoming**
Just to make sure everyone knows when making a PR

@khangmach @carinediaz you have pushed the "Watch" button, right? :) Just asking (I often forget that I have unwatched repos when taking breaks etc)

### CODE REVIEW

**Comment by jamalex** [comment]
Doesn't look like they have, no.

@carinediaz @khangmach Please toggle to "Watching":

image

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This is the code for Learning Equality's homepage: [https://learningequality.org](https://learningequality.org)
 
+Short URL for the site is [http://leq.ngo](http://leq.ngo)
+
+ping @carinediaz for text changes and @khangmach for technical changes and deployment of changes
```
Comment (indirectlylit) [inline README.md]: let's make this:

> Ping @carinediaz to review content changes and @khangmach to review design changes.

As far as reviewing technical changes and deploying, that's mostly on the engineering side

Comment (benjaoming) [reply README.md]: Oh so that would raise another question: Who to ping for deploying stuff? :)

Comment (indirectlylit) [reply README.md]: right now it's just the #sysops room ... aron, jamie, ivan, and soon me? could you, too! ;)

Comment (khangmach) [reply README.md]: 👍

Comment (carinediaz) [reply README.md]: thanks for I had no idea what was going on. Also, FYI will be reviewing all of those tomorrow, when hopefully I'll be less under the weather and with more bandwidth. Quite literally.

**Pull request merged**